### DISCUSSION TITLE
Removing border about nvim tree mappings popup.

### DISCUSSION THREAD

**Discussion by croyleje** [nvim-tree/nvim-tree.lua#2748] [Q&A]
I have been trying to find a way to remove the border around the nvim tree mapping popup or prompt.  The default binding is "?".  If anyone is aware of a way to either remove the border or is there is a highlight group I can use to change the color it would be greatly appreciated.

Thank you,
Jason

**Comment by alex-courtis** [comment]
Unfortunately we don't have any user customisation options for the help.

We do apply three highlights: https://github.com/nvim-tree/nvim-tree.lua/blob/39e6fef85ac3bb29532b877aa7c9c34911c661af/lua/nvim-tree/help.lua

However they cannot be configured separately from nvim-tree.

If you'd like to [submit a PR](https://github.com/nvim-tree/nvim-tree.lua/blob/master/CONTRIBUTING.md) to add this functionality...

**Reply by croyleje** [reply]
Sounds like a great idea. I will see what I can do with it tonight have a wonderful day and thank for the quick response and helpful information.
